users/ignoring.rst

```diff
@@ -44,7 +44,7 @@ The ``.stignore`` file contains a list of file or path patterns. The
    ``/foo`` matches ``foo`` but not ``subdir/foo``.
 
 -  A pattern beginning with ``#include`` results in loading patterns
-   from the named file. It is an error for a file to not exist or be
+   from the named file which must reside in the repository. It is an error for a file to not exist or be
```

Comment:
This was actually not the case, it's just that the name is always relative to the folder root, no? So I think you can reach c:\ignores.txt from c:\stuff\sync by `#include ../../ignores.txt`, no?